qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -263,11 +263,11 @@
       - Targeted connectivity
     * - ``"rustiq"``
       - :class:`~.PauliEvolutionSynthesisRustiq`
-      - use a diagonalizing Clifford per Pauli term
+      - use the synthesis method from https://github.com/smartiel/rustiq-core
       - all-to-all
```

Comment:
I am not 100% sure that this is the right way to include a link. Should I do something like `<...>__`?